packages/hadron-ipc/index.js

```diff
@@ -2,8 +2,19 @@
 
 const isRenderer = require('is-electron-renderer');
 
-if (isRenderer) {
-  module.exports = require('./lib/renderer');
-} else {
-  module.exports = require('./lib/main');
+const isElectron = typeof require('electron') !== 'string';
```

Comment:
This assumes that `electron` is always present as a peer dependency, which it might not be.

Reply:
```suggestion
let isElectron = false;
try {
  isElectron = typeof require('electron') !== 'string';
} catch { /* not electron */ }
```